objectivec/Tests/GPBMessageTests+Serialization.m

```diff
@@ -989,8 +989,8 @@ - (void)testMap_CorruptedWireFormat {
   TestMap *msg = [TestMap parseFromData:data error:&error];
   XCTAssertNil(msg);
   XCTAssertNotNil(error);
-  XCTAssertEqualObjects(error.domain, GPBMessageErrorDomain);
-  XCTAssertEqual(error.code, GPBMessageErrorCodeMalformedData);
+  XCTAssertEqualObjects(error.domain, GPBCodedInputStreamErrorDomain);
+  XCTAssertEqual(error.code, GPBCodedInputStreamErrorInvalidSubsectionLimit);
```

Comment:
We should probably do test to force all the errors to ensure they are coming through correctly.